pos_payment_terminal/static/src/xml/pos_payment_terminal.xml

```diff
@@ -3,7 +3,7 @@
     <t t-extend="PaymentScreen-Paymentlines" >
         <t t-jquery=".col-name" t-operation="append">
             <t t-if="line.cashregister.journal.payment_mode and widget.pos.config.iface_payment_terminal">
-                   <button class="payment-terminal-transaction-start"  t-att-data-cid='line.cid'>Start transaction</button>
+                   <button class="payment-terminal-transaction-start"  t-att-data-cid='line.cid'>Send to terminal</button>
```

Comment:
Why this change? IMHO user 'functionnaly' understands more 'Start transaction' than send one. But it's my feeling

Reply:
I thought it was easier to understand but this is only my point of view. Not changing it should be better for the users in case of migrations